vector/overlays/moc/zero/opf-observatorium/service.yaml

```diff
@@ -0,0 +1,17 @@
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: vector-metrics
+spec:
+  selector:
+    app: vector
+  ports:
+    - name: metrics
+      protocol: TCP
+      port: 80
+      targetPort: metrics
+    - name: graphql-api
+      protocol: TCP
+      port: 8000
```

Comment:
Is there a reason to remap the port numbers? Why not keep it consistent with the actually exposed ports?

Reply:
updated :+1: